device-types/SNR/S2995G-24FX.yaml

```diff
@@ -2,62 +2,108 @@
 manufacturer: SNR
 model: S2995G-24FX
 slug: snr-s2995g-24fx
-part_number: S2995G-24FX-UPS
+part_number: SNR-S2995G-24FX
 u_height: 1.0
 is_full_depth: false
-weight: 4.1
+weight: 4.06
 weight_unit: kg
 airflow: right-to-left
 comments: '[SNR-S2995G-24FX](https://snr.systems/ru/catalog/setevoe-oborudovanie/kommutatory-agregacii-i-yadra/snr-s2995g/snr-s2995g-24fx)'
 console-ports:
   - name: con0
-    label: Console
+    label: CONSOLE
     type: rj-45
 power-ports:
-  - name: PSU1
+  - name: power AC
     type: iec-60320-c14
-    maximum_draw: 80
+    maximum_draw: 60
+    label: 100-240V; 50/60Hz; 1.0A
+  - name: power DC
+    type: other
+    maximum_draw: 60
+    label: DC 11-13V, 4.5A
+power-outlets:
+  - name: usb
```

Comment:
USB-a ports as power-outlets is not something currently being modeled in this repo